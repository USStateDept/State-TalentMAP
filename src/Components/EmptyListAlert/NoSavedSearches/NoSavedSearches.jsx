import React from 'react';
import EmptyListAlert from '../EmptyListAlert';

const NoSavedSearches = () => {
  const textLineTwo = `Reuse this job search in the future by clicking "Save this search".
  Try naming your search something memorable, like "Summer Positions 2018".`;
  return (
    <EmptyListAlert
      textLineOne="You haven't saved any searches."
      textLineTwo={
      (
        <span>{textLineTwo}</span>
      )
    }
    />
  );
};

export default NoSavedSearches;
