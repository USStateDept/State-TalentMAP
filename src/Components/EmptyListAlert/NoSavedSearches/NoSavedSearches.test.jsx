import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NoSavedSearches from './NoSavedSearches';

describe('NoSavedSearchesComponent', () => {
  it('is defined', () => {
    const wrapper = shallow(<NoSavedSearches />);
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<NoSavedSearches />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
