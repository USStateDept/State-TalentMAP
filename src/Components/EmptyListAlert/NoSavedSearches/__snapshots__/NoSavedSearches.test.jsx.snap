// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSavedSearchesComponent matches snapshot 1`] = `
<EmptyListAlert
  textLineOne="You haven't saved any searches."
  textLineTwo={
    <span>
      Reuse this job search in the future by clicking "Save this search".
  Try naming your search something memorable, like "Summer Positions 2018".
    </span>
  }
/>
`;
